cmd/virt-prerunner/main.go

```diff
@@ -30,8 +31,10 @@ import (
 func main() {
 	var vmData string
 	var receiveMigration bool
+	extraVFIOMemoryLockSize := resource.QuantityValue{Quantity: resource.MustParse("1Gi")}
```

Comment:
Why is this required?

Reply:
For VFIO device passthouth, whole memrory lock is needed. and extra 1Gi is an empirical value from [libvirt](
 https://gitlab.com/libvirt/libvirt/-/blob/master/src/qemu/qemu_domain.c#L9350).

Reply:
So this is unrelated to this issue? It deserves another patch.

Reply:
a `privileged` process can [lock any amount of memory](https://linux.die.net/man/2/prlimit). but an unprivileged process default can only lock a small amount of memory(`prlimit -l`). now we use `privileged=false` pod, we have to adjust this limit, and this is why we add `CAP_SYS_RESOURCE` capability.